bot/bot.py

```diff
@@ -368,6 +366,38 @@ async def error_handle(update: Update, context: CallbackContext) -> None:
     except:
         await context.bot.send_message(update.effective_chat.id, "Some error in error handler")
 
+
+async def error_handle_except_network(update: Update, context: CallbackContext) -> None:
```

Comment:
I don't understand what does it has to do with the support of the sqlite db. Interesting addition although

Reply:
fixed!